care/facility/models/patient_base.py

```diff
@@ -73,12 +73,15 @@ def reverse_choices(choices):
     ("Critical", "Critical"),
 ]
 
-DISCHARGE_REASON_CHOICES = [
-    ("REC", "Recovered"),
-    ("REF", "Referred"),
-    ("EXP", "Expired"),
-    ("LAMA", "LAMA"),
-]
+
+class DiseaseReasonEnum(enum.IntEnum):
```

Comment:
use `models.IntegerChoices` in place of base enum, as it will generate choices automatically, and you can reference the value directly without `.value`

[ref](https://docs.djangoproject.com/en/5.0/ref/models/fields/#enumeration-types)